### PR TITLE
Fix search issue on list policy view

### PR DESCRIPTION
**Pull request by GowthamShanmugam**
BZ: https://bugzilla.redhat.com/show_bug.cgi?id=2233445

### CODE REVIEW

**Comment by GowthamShanmugam** [comment]
Before Fix:
Search_Box

**Comment by GowthamShanmugam** [comment]
After fix:
image

File: packages/mco/components/modals/app-manage-policies/policy-list-view.tsx [hunk 1]
```diff
@@ -43,6 +43,18 @@ const filterPolicies = (dataPolicyInfo: DataPolicyType[], searchText: string) =>
     getName(policy).toLowerCase().includes(searchText)
   );
 
+// **Note: PatternFly change the fn signature
+// From: (value: string, event: React.FormEvent<HTMLInputElement>) => void
+// To: (_event: React.FormEvent<HTMLInputElement>, value: string) => void
+// both cases need to be handled for backwards compatibility
+const onChange = (input: any) => {
```
Comment (bipuladh) [inline packages/mco/components/modals/app-manage-policies/policy-list-view.tsx:50]:
```suggestion
const onChange = (input: unknown) => {
```

Comment (SanjalKatiyar) [reply packages/mco/components/modals/app-manage-policies/policy-list-view.tsx:50]: can we try `string | HTMLInputElement` ??

Comment (SanjalKatiyar) [reply packages/mco/components/modals/app-manage-policies/policy-list-view.tsx:50]: can we try `string | HTMLInputElement` ??

Comment (GowthamShanmugam) [reply packages/mco/components/modals/app-manage-policies/policy-list-view.tsx:50]: ack

**Pull request merged**